core/services/directrequestocr/orm.go

```diff
@@ -9,13 +9,14 @@ import (
 	"github.com/smartcontractkit/sqlx"
 
 	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr/config"
 	"github.com/smartcontractkit/chainlink/core/services/pg"
 )
 
 //go:generate mockery --quiet --name ORM --output ./mocks/ --case=underscore
 
 type ORM interface {
-	CreateRequest(requestID RequestID, receivedAt time.Time, requestTxHash *common.Hash, qopts ...pg.QOpt) error
+	CreateRequest(requestID RequestID, receivedAt time.Time, requestTxHash *common.Hash, aggregationMethod config.AggregationMethod, qopts ...pg.QOpt) error
```

Comment:
We're gonna add the commitment right after that so I wonder if it makes sense to also wrap it all in a nested "metadata" struct just like we're planning to do in the contract?